students/PeterB/Lesson09/CLI.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python3
 
 import sys
+from Lesson9mailboxmoneyOO import Donor
```

Comment:
Since you're importing as Donor and DonorDB, you dont need to prepend "d" before classing them